### PR TITLE
[PUL-117] Possible fix for form.end spacing

### PR DESCRIPTION
**Pull request by james-manley**

### CODE REVIEW

**Comment by Stanton** [comment]
Addresses #44

File: views/pulsar/v2/helpers/form.html.twig [hunk 1]
```diff
@@ -573,7 +573,6 @@ class   | string | A space separated list of class names
 
 #}
 {% macro end(options = {}) %}
-{% spaceless %}
```
Comment (Stanton) [inline views/pulsar/v2/helpers/form.html.twig]: Wouldn't've thought removing the top level `spaceless` was required? Didn't removing the whitespace control on L:588 resolve the issue alone?

Comment (james-manley) [inline views/pulsar/v2/helpers/form.html.twig]: I'd have thought so too, but seems on a combination of removing both solves it. I'm no expert in Twig stuff though!

Comment (Stanton) [inline views/pulsar/v2/helpers/form.html.twig]: Yep, you're right!

**Pull request merged**